src/test/java/fi/otavanopisto/kuntaapi/test/AbstractTest.java

```diff
@@ -121,6 +141,18 @@ protected long executeInsert(String sql, Object... params) {
     
     return -1;
   }
+
+  private void applyStatementParams(Connection connection, PreparedStatement statement, Object... params)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "connection". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)